### ISSUE TITLE
using softmax at very last layer

### ISSUE THREAD

**Issue by gihyunkim** [ronghuaiyang/arcface-pytorch#66]
paper show the architecture of arcface and it has softmax function at last layer. But seems like u don't use it on your model. 

what is difference whether use softmax or not? I am confused someone use softmax and others don't.

**Comment by LeeBC2298** [comment]
In fact, torch.nn.CrossEntropyLoss has softmax opetation.
torch.nn.NLLLoss is the real 'cross entropy loss'.
